### DISCUSSION TITLE
Section 1: Transition from core package to ecosystem?

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#68] [Text comments] [answered]
> In particular, the transition from focusing our energy on development and maintenance
of a single core package, to instead maintaining the core package and fostering the
development of the community and its expansion has been a key goal of the Astropy
Project over the last several years.

Personally, I don't see that as a "transition" because it's not an either or. Plenty of people still do most of their work in the core package and a lot of energy goes into "development and maintenace" of that package, so "our" is a good word, since it implies that this transition is done by all authors. I have not run the math, but I would not be surprised if there are more issues and PRs to core than all other packages combined.

I would prefer a wording that says something like "additional components have been added to project goals" etc., not something that could be interpreted as "the project took a vote and decided that the core package is now of secondary interest".

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
